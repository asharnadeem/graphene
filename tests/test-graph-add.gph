int main(){
    graph<int> g;
    g.add(0, 2);
    g.add(1, 3);
    print(g[0]);
    print(g[1]);
}