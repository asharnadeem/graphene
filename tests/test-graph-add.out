id: 0, val: 2, edges: 0
id: 1, val: 3, edges: 0
